dockerfiles/RPCDockerfile

```diff
@@ -19,4 +19,6 @@ COPY monitoring/metrics_config.yaml /container/
 
 ENV CLIPPER_MODEL_PATH=/model
 
+HEALTHCHECK --interval=3s --timeout=3s --retries=1 CMD test -f /model_is_ready.check || exit 1
```

Comment:
What is the `retries` field for?

Reply:
"It takes <retries> consecutive failures of the health check for the container to be considered `unhealthy`"